src/context/datasetContext/index.tsx

```diff
@@ -27,9 +27,10 @@ import {
 } from "@inrupt/solid-client";
 
 import useDataset from "../../hooks/useDataset";
+import { WithAcp } from "@inrupt/solid-client/dist/acp/acp";
```

Comment:
This needs to be changed to the following, afaik:
```suggestion
import { WithAcp } from "@inrupt/solid-client/acp/acp";
```